26/Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM openjdk:8-jdk
+MAINTAINER Maychell Oliveira <maychell.oliveira@codeminer42.com>
+
+# Android SDK
+ENV ANDROID_HOME /srv/var/android-sdk
+
+# Install required tools
+RUN apt-get --quiet update --yes \
+    && apt-get --quiet install --yes wget tar unzip lib32stdc++6 lib32z1 --no-install-recommends
+
+# Install Android SDK tools into ${ANDROID_HOME}
+RUN mkdir -p ${ANDROID_HOME}
+RUN wget --quiet --output-document=android-sdk.zip https://dl.google.com/android/repository/sdk-tools-linux-4333796.zip \
+    && unzip android-sdk.zip -d ${ANDROID_HOME} \
+    && rm -f android-sdk.zip
+
+ENV PATH ${PATH}:${ANDROID_HOME}/tools:${ANDROID_HOME}/tools/bin:${ANDROID_HOME}/platform-tools
```

Comment:
Couldn't you create a link to the binary instead of override the PATH? (Like you did on line 42)